mods/cnc/maps/nod04b/nod04b.lua

```diff
@@ -32,10 +32,17 @@ WorldLoaded = function()
 	GDI = Player.GetPlayer("GDI")
 
 	Trigger.AfterDelay(DateTime.Seconds(3), function()
-		Reinforcements.ReinforceWithTransport(GDI, "apc", Apc1Units, Apc1Waypoints, nil,
-			function(transport, cargo)
-				Utils.Do(cargo, IdleHunt)
```

Comment:
This removes passenger unloading. Is this an accident? The commit message doesn't mention it.